Dockerfile

```diff
@@ -1,8 +1,6 @@
 # Use OpenShift golang builder image
 # These images needs to be synced with the images in the Makefile.
-ARG BUILDER_IMAGE=${BUILDER_IMAGE:-registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17}
-ARG TARGET_IMAGE=${TARGET_IMAGE:-registry.ci.openshift.org/ocp/4.17:base-rhel9}
-FROM ${BUILDER_IMAGE} AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.22.7 AS builder
```

Comment:
`BUILDER_IMAGE` and `TARGET_IMAGE` should also be dropped from the Makefile and the documentation.